public/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Create React App Sample",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "newFavicon.png",
```

Comment:
- You need to have `.ico` extension if you are adding the image as fav icon

Reply:
- Please Add screen shot of the image which you will like to add

Reply:
ohh okay !! sure